examples/kebechet/alerting.md

```diff
@@ -0,0 +1,29 @@
+# Alerting
+
+Once we have all metrics collected and stored, we can define the alerts that are going to be triggered. These alerts can be provided by:
+
+- Github Issues, using Prometheus GitHub AlertManager.
```

Comment:
can we add a description to what the GitHub Alertmanager is and link to: https://github.com/m-lab/alertmanager-github-receiver? You can also add "To learn more about setting up the GitHub Alertmanager you can refer to our documentation: https://www.operate-first.cloud/operations/sre/incident-management/github-receiver-setup.md"

Reply:
Great!